setup.py

```diff
@@ -147,7 +147,10 @@
 
 SINGLESTORE_REQUIRED = ["singlestoredb<1.8.0"]
 
-COUCHBASE_REQUIRED = ["couchbase==4.3.2"]
+COUCHBASE_REQUIRED = [
+    "couchbase==4.3.2",
+    "couchbase-columnar==1.0.0"
```

Comment:
Add this in https://github.com/feast-dev/feast/blob/master/pyproject.toml#L58 as well

Reply:
Thanks, added: [645467b](https://github.com/feast-dev/feast/pull/5025/commits/645467bdae99a0aa14b8a9077ca72f80abfc7680)